src/warehouses/components/WarehouseSettings/WarehouseSettings.tsx

```diff
@@ -0,0 +1,200 @@
+import { Card, CardContent, Divider, Typography } from "@material-ui/core";
+import CardSpacer from "@saleor/components/CardSpacer";
+import CardTitle from "@saleor/components/CardTitle";
+import { FormSpacer } from "@saleor/components/FormSpacer";
+import Link from "@saleor/components/Link";
+import { RadioGroupField } from "@saleor/components/RadioGroupField";
+import Skeleton from "@saleor/components/Skeleton";
+import { makeStyles } from "@saleor/macaw-ui";
+import { renderCollection } from "@saleor/misc";
+import { WarehouseDetails_warehouse_shippingZones_edges_node } from "@saleor/warehouses/types/WarehouseDetails";
+import React from "react";
+import { FormattedMessage, useIntl } from "react-intl";
+
+import { WarehouseClickAndCollectOptionEnum } from "./../../../types/globalTypes";
+import { WarehouseDetailsPageFormData } from "./../WarehouseDetailsPage";
+
+export interface WarehouseInfoProps {
+  zones: WarehouseDetails_warehouse_shippingZones_edges_node[];
+  disabled: boolean;
+  data: WarehouseDetailsPageFormData;
+  onShippingZoneClick: (id: string) => void;
+  onChange: (event: React.ChangeEvent<any>) => void;
+  setData: (data: Partial<WarehouseDetailsPageFormData>) => void;
+}
+
+const useStyles = makeStyles(
+  theme => ({
+    link: {
+      "&:not(:last-of-type)": {
+        marginBottom: theme.spacing()
+      }
+    }
+  }),
+  {
+    name: "WarehouseInfoProps"
+  }
+);
+
+const WarehouseInfo: React.FC<WarehouseInfoProps> = ({
```

Comment:
Component is called `WarehouseInfo` but sits inside a `WarehouseSettings` file name, the component <-> filename should be identical

Reply:
Done, I have also changed the name of the interface `WarehouseInfoProps` -> `WarehouseSettingsProps`